src/JWT/JWT.csproj

```diff
@@ -36,8 +36,16 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <DefineConstants>SYSTEMTEXTJSON;$(DefineConstants)</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="System.Text.Json" Version="4.7.2" />
+  </ItemGroup>
+
   <ItemGroup>
-   <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
```

Comment:
Should we reference either one or another?

Reply:
And the another, already existing serializer must be included only if the target framework is lower than netstd2

Reply:
I don't see any problem in referencing Newtonsoft.Json in netstandard20 ... 
I simply like the option to use System.Text.Json ... 
maybe in the future with net50 things can be different and we can drop support for Newtonsoft.Json